docs/examples/Serialization Example.ipynb

```diff
@@ -18,13 +18,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 3,
```

Comment:
please restart kernel, clear output and rerun again to set the execution count back to `1`